lib/kafo/logging.rb

```diff
@@ -69,8 +52,33 @@ def setup_file_logging(log_level, log_dir, log_owner, log_group)
         end
       end
 
+      def set_color_scheme
+        ::Logging.color_scheme(
+          'bright',
+          :levels => {
+            :info  => :green,
+            :warn  => :yellow,
+            :error => :red,
+            :fatal => [:white, :on_red]
+          },
+          :date   => :blue,
+          :logger => :cyan,
+          :line   => :yellow,
+          :file   => :yellow,
+          :method => :yellow
+        )
+      end
+
+      def layout(color = false)
```

Comment:
Nit: I think that this is more generic than it needs to be, and the `color_layout` wrapper below is therefore not necessary. You could eliminate the wrapper by calling `KafoConfigure.use_colors?` directly inside `layout` instead of using the `colors` parameter, and then simply use `:layout => layout,` on L#89

Reply:
This is the spot that led to this design -- https://github.com/theforeman/kafo/pull/276/files#diff-d93616df475bf3004549b979a7abd583R41

I need to be able to explicitly get the layout without colors and be able to choose a color layout in the verbose case based on the configuration coming from Kafo.

Reply:
I will think on some further re-factoring to simplify this.

Reply:
OK, that makes sense. I took a stab at it with #279 to see what it might look like and you can let me know if you think it's worth doing.